gradle.properties

```diff
@@ -0,0 +1,6 @@
+//signing.keyId=<publicKey>
```

Comment:
The settings in `gradle.properties` are only required when we run the `uploadArchives` task. Most of the time there is no need for the file to exist (unless one needs to configure stuff like we`b proxy settings to download gradle or artefacts from Maven Central). 

I think the simplest option is to add the file into `.gitignore` then and leave the existing checked in version an an illustration of what needs to be set in it when uploading. 

What do you think?